nep-11.mediawiki

```diff
@@ -0,0 +1,418 @@
+    {
+      "name": "to",
+      "type": "Hash160"
+    },
+    {
+      "name": "amount",
+      "type": "Integer"
+    },
+    {
+      "name": "tokenid",
+      "type": "ByteString"
+    }
+  ],
+  "returntype": "Boolean"
+}
+</pre>
+
+It transfers an <code>amount</code> of tokens with id <code>tokenid</code> from address <code>from</code> to address <code>to</code>.
+
+The parameters <code>from</code> and <code>to</code> SHOULD be 20-byte addresses.
+If not, this method SHOULD <code>throw</code> an exception.
+
+The parameter <code>amount</code> SHOULD be greater than or equal to <code>0</code> and SHOULD be less than or equal to <code>pow(10, decimals())</code>.
+If not, this method SHOULD <code>throw</code> an exception.
+
+The parameter <code>tokenid</code> SHOULD be a valid NFT.
+If not, this method SHOULD <code>throw</code> an exception.
+
+The function SHOULD return <code>false</code> if the <code>from</code> account balance does not have enough tokens to spend.
+
+If the method succeeds, it MUST fire the <code>Transfer</code> event, and MUST return <code>true</code>, even if the <code>amount</code> is <code>0</code>, or the token is sent to the owner.
+
+If the receiver is a deployed contract, the function MUST call <code>onNEP11Payment</code> method on receiver contract AFTER firing the <code>Transfer</code> event.
+
+The function SHOULD check whether the <code>from</code> address equals the caller contract hash.
+If so, the transfer SHOULD be processed; If not, the function SHOULD use the SYSCALL <code>Neo.Runtime.CheckWitness</code> to verify the transfer.
+
+If the transfer is not processed, the function SHOULD return <code>false</code>.
+
+====ownerOf====
+
+<pre>
+{
+  "name": "ownerOf",
+  "parameters": [
+    {
+      "name": "tokenid",
+      "type": "ByteString"
+    }
+  ],
+  "returntype": "InteropInterface<iterator>"
+}
+</pre>
+
+Returns an <code>iterator</code> that contains all the co-owners that own the specified token.
+
+The parameter <code>tokenid</code> SHOULD be a valid NFT.
+If not, this method SHOULD <code>throw</code> an exception.
+
+====balanceOf====
+
+<pre>
+{
+  "name": "balanceOf",
+  "parameters": [
+    {
+      "name": "owner",
+      "type": "Hash160"
+    },
+    {
+      "name": "tokenid",
+      "type": "ByteString"
+    }
+  ],
+  "returntype": "Integer"
+}
+</pre>
+
+This method returns the balance of the specified token for the specified <code>owner</code>'s account.
+
+The parameter <code>tokenid</code> SHOULD be a valid NFT.
+If not, this method SHOULD <code>throw</code> an exception.
+
+The parameter <code>owner</code> SHOULD be a 20-byte address.
+If not, this method SHOULD <code>throw</code> an exception.
+
+If the <code>owner</code> is an unused address, or it's not the owner of the specified token, this method SHOULD return <code>0</code>.
+
+===Optional methods===
+
+====tokens====
+
+<pre>
+{
+  "name": "tokens",
+  "parameters": [],
+  "returntype": "InteropInterface<iterator>"
```

Comment:
The type is currently `Any` in C# node, I think it is better to change manifest in C# node, to be more specific.

Reply:
It should be fixed in neo-project/neo#2356